collection-cacheable-for-spring-starter/src/main/java/de/qaware/tools/collectioncacheableforspring/creator/SetCollectionCreator.java

```diff
@@ -0,0 +1,40 @@
+/*-
+ * #%L
+ * Collection Cacheable for Spring :: Starter
+ * %%
+ * Copyright (C) 2020 QAware GmbH
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+package de.qaware.tools.collectioncacheableforspring.creator;
+
+import org.springframework.stereotype.Component;
+
+import java.util.Collection;
+import java.util.HashSet;
+
+@Component
+public class SetCollectionCreator
+        implements CollectionCreator {
+    @Override
+    public boolean canHandle(Class<?> cls) {
+        return cls.isAssignableFrom(HashSet.class);
```

Comment:
Shouldn't this be `equals`? What happens if somebody uses an argument with type which derives from HashSet? Can you add a test case for that?

Reply:
Alright, thanks for clarification and the test case! I'll merge it and release.